From 36dfffebed3b9c0f43df2717fa275b9373e9b442 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Tue, 10 Apr 2018 23:08:39 +0200
Subject: [PATCH 1/1] vexpress: fix syntax error in armv7_boot_nonsec_default()

With CONFIG_ARMV7_BOOT_SEC_DEFAULT=y a syntax error occurs.
Add the missing semicolon.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 board/armltd/vexpress/vexpress_tc2.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/board/armltd/vexpress/vexpress_tc2.c b/board/armltd/vexpress/vexpress_tc2.c
index b143e04097..9cd0ec8ea0 100644
--- a/board/armltd/vexpress/vexpress_tc2.c
+++ b/board/armltd/vexpress/vexpress_tc2.c
@@ -18,7 +18,7 @@
 bool armv7_boot_nonsec_default(void)
 {
 #ifdef CONFIG_ARMV7_BOOT_SEC_DEFAULT
-	return false
+	return false;
 #else
 	/*
 	 * The Serial Configuration Controller (SCC) register at address 0x700
-- 
2.14.2

